.claude/commands/create-pr.md

```
@@ -0,0 +1,81 @@
+---
+name: create-pr
+description: Rebase from the latest origin/main, squash commits, and create a PR on GitHub with intelligent commit messages. Invoke with /create-pr.
+---
+
+# Create Pull Request
+
+Rebase from the latest `origin/main`, squash commits, and create a PR on GitHub.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)